repository/process.go

```diff
@@ -69,8 +60,11 @@ func ProcessRepos(gitxargsConfig *config.GitXargsConfig, repos []*github.Reposit
 					"Repo name": repo.GetName(), "Error": processErr,
 				}).Debug("Error encountered while processing repo")
 			}
-			return processErr
 
+			p.Increment()
+			wg.Done()
```

Comment:
NIT: probably better to do a `defer` at the top, although effectively this is the same. `defer` is just more defensive coding if this logic is ever extended.